democracy_club/templates/hermes/post_detail.html

```diff
@@ -32,6 +32,14 @@ <h1>{{ post.subject }}</h1>
                     {% endfor %}
                 </address>
                 on <time pubdate datetime="{{post.modified_on|date:"c"}}" title="{{post.created_on|date:"jS E Y"}}">{{post.created_on|date:"jS E Y"}}</time>
+                {% if post.tags %}
+                    <span aria-hidden="true">🏷️</span>
+                    {% for tag in post.tags %}
+                        <ul>
```

Comment:
`ul` needs to be outside the loop (drive by review!)

Reply:
Fixed in https://github.com/DemocracyClub/Website/pull/739